crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/prosecution_concluded/scheduler/ProsecutionConcludedScheduler.java

```diff
@@ -31,12 +31,12 @@
 @ConditionalOnProperty(value = "feature.prosecution-concluded-schedule.enabled", havingValue = "true")
 public class ProsecutionConcludedScheduler {
 
-    private final ProsecutionConcludedRepository prosecutionConcludedRepository;
-    private final ProsecutionConcludedService prosecutionConcludedService;
-    private final MaatCourtDataService maatCourtDataService;
     private final ObjectMapper objectMapper;
+    private final CourtDataAPIService courtDataAPIService;
+    private final ProsecutionConcludedService prosecutionConcludedService;
+    private final ProsecutionConcludedRepository prosecutionConcludedRepository;
 
-    @Scheduled(cron = "${queue.message.log.cron.expression}")
+    @Scheduled(fixedRate = 5000)
```

Comment:
Shall we keep the values in application.yml ?

Reply:
Oops, that definitely shouldn't have been changed!